package.json

```diff
@@ -108,6 +108,8 @@
     "openapi-fetch": "^0.12.5",
     "react-aria": "^3.39.0",
     "react-aria-components": "^1.8.0",
+    "react-dnd": "^16.0.1",
+    "react-dnd-html5-backend": "^16.0.1",
```

Comment:
Can we add these to the `no-restricted-imports` eslint rule in `eslint.config.mjs` so we can restrict use to the UI directory?